enterprise/frontend/src/embedding-sdk/cli/constants/messages.ts

```diff
@@ -27,13 +27,19 @@ export const EMBEDDING_FAILED_MESSAGE = `
   ${DELETE_CONTAINER_MESSAGE}
 `;
 
-// eslint-disable-next-line no-unconditional-metabase-links-render -- link for the CLI message
-export const METABASE_INSTANCE_SETUP_COMPLETE_MESSAGE = `
+export const PREMIUM_TOKEN_REQUIRED_MESSAGE =
+  "  Don't forget to add your premium token to your Metabase instance in the admin settings! The embedding demo will not work without a license.";
+
+export const getMetabaseInstanceSetupCompleteMessage = (instanceUrl: string) =>
+  // eslint-disable-next-line no-unconditional-metabase-links-render -- link for the CLI message
+  `
   Metabase instance is ready for embedding.
+  Go to ${instanceUrl} to start using Metabase.
```

Comment:
I find myself re-typing the instance url often, so it's nice to show them at the end of the setup.